specification/maps/resource-manager/Microsoft.Maps/stable/2021-02-01/maps-management.json

```diff
@@ -0,0 +1,1105 @@
+        "x-ms-pageable": {
+          "nextLinkName": null,
+          "itemName": "value"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/CreatorList"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Maps/accounts/{accountName}/creators/{creatorName}": {
+      "put": {
+        "tags": [
+          "Creators"
+        ],
+        "description": "Create or update a Maps Creator resource. Creator resource will manage Azure resources required to populate a custom set of mapping data. It requires an account to exist before it can be created.",
+        "operationId": "Creators_CreateOrUpdate",
+        "x-ms-examples": {
+          "Create Creator Resource": {
+            "$ref": "./examples/CreateMapsCreator.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CreatorNameParameter"
+          },
+          {
+            "name": "CreatorResource",
+            "in": "body",
+            "description": "The new or updated parameters for the Creator resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Creator"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The Creator will be created asynchronously. The Creator will be ready to use once the provisioningState property changed from 'Created' to 'Succeeded' through creators Get API.",
```

Comment:
you should still support a 200 in case of no-op PUTs for example, or PUTs to update tags or some such that can be done snchronously

Reply:
What is a no-op PUTs? I thought update tags is through patch API instead of PUT.

Reply:
I updated and forwarded this to the team.